templates/report_template.html

```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
 <html lang="en">
+	<img src="{{summary.chart_url}}" width="500" />
```

Comment:
I think this would be better suited near the bottom with the summary statistics

Reply:
I had the same dilemma, but I thought the entire email gets clipped so this would be better up top for people not to go all the way down to check the summary, but i'll wait for more votes for the bottom, i can move it

Reply:
Alternatively we could move the summary to the top - I'm fine either way as long as they are together